install/kubernetes/cilium/files/nodeinit/prestop.bash

```diff
@@ -22,8 +22,8 @@ if ip link show cilium_host; then
   ip link del cilium_host
 fi
 
-{{- if not (eq .Values.nodeinit.bootstrapFile "") }}
-rm -f {{ .Values.nodeinit.bootstrapFile }}
+{{- if not (eq .Values.nodeinit.bootstrapDir "") }}
```

Comment:
[This can be simplified](https://helm.sh/docs/chart_template_guide/control_structures/#ifelse):
```suggestion
{{- if .Values.nodeinit.bootstrapDir }}
```
The only reason to do the explicit check is in case you want to delete the directory when the value is `null`. Otherwise the above check should work.